setup.py

```diff
@@ -24,6 +24,7 @@
     's3': ['boto3 >= 1.4.7'],
     'zookeeper': ['kazoo >= 2.6.1'],
     'jupyter-metrics': ['plotly', 'ipyplot', 'jupyter-dash'],
+    'clearml': ['clearml', 'pandas']
 }
 EXTRAS_REQUIRE['all'] = sum((deps for env, deps in EXTRAS_REQUIRE.items() if env != 'dev'), [])
```

Comment:
Let's not include integrations in `all` variable: I suggest to create separate `integrations` variable with all future integrations